### DISCUSSION TITLE
Enable Web Search by Default for Custom Model Deployment

### DISCUSSION THREAD

**Discussion by Tavernari** [open-webui/open-webui#5592] [General]
**Is your feature request related to a problem? Please describe.**
I am working on deploying a custom model and would like to enable web search functionality by default. Currently, there is no straightforward way to automatically enable the web-search plugin when launching the model.

**Describe the solution you'd like**
I propose adding an option to configure the model to have the web-search plugin enabled by default upon initialization. This would streamline the user experience and reduce the need for manual intervention to activate web search.

**Describe alternatives you've considered**
At the moment, I have not found any viable alternative solutions for enabling the web-search plugin automatically.

**Additional context**
Having web search enabled by default would enhance the model's ability to provide real-time, up-to-date information, making it more useful in scenarios where current data is critical.

**Comment by tjbck** [comment]
Would using `/?web-search=true` url param cover your use case?

**Comment by 1e100** [comment]
+1. Would really like to see this feature, particularly if this can be set on a per-model basis. GPT OSS with search is much better than without.
